migrator/migrations/m_211_to_m_212_admission_control_config/store/store.go

```diff
@@ -0,0 +1,101 @@
+package store
```

Comment:
nit:  I would prefer this be in `m_211_to_m_212_admission_control_config/test/store`